Signum.Engine.Extensions/Workflow/CaseActivityLogic.cs

```diff
@@ -337,6 +348,23 @@ select new
             }
         }
 
+        public static void RegisterLastCaseActivityExpression(SchemaBuilder sb)
+        {
+            var types = Schema.Current.Tables.Keys.Where(a => typeof(ICaseMainEntity).IsAssignableFrom(a));
+
+            foreach (var type in types)
+            {
+                giRegisterLastCaseActivityExpression.GetInvoker(type)(sb);
+            }
+        }
+

```

Comment:
I think this is not necessary, you can register the expression directly on `ICaseMAinEntity`